### PR TITLE
Refactor Slider

### PR DESCRIPTION
**Pull request by max-voronov**
In this PR the slider component is being converted into a functional component
Please review :)

### CODE REVIEW

File: src/components/Slider.js [hunk 1]
```diff
@@ -1,39 +1,25 @@
...
-  static getDerivedStateFromProps(nextProps, prevState) {
-    // if the keyboards are changing the volume, adjust slider accordingly
-    if (nextProps.currentVolume * 10 !== prevState.sliderVal) {
-      return { sliderVal: nextProps.currentVolume * 10 };
-    }
-    return null;
-  }
-
-  handleSliderChange(e) {
+function Slider({ currentVolume, setTargetVolume }) {
```
Comment (huyenltnguyen) [inline src/components/Slider.js:4]: Nit: Could we make this an array function, since we are using ES6 in other places 😄

**Review by huyenltnguyen — COMMENTED** [review]
I just have a small nit regarding the function syntax. Otherwise, the PR looks good to me

File: src/components/Slider.js (hunk 1 above)
Comment (ahmadabdolsaheb) [reply src/components/Slider.js:4]: other than that it is LGMT 👍

Comment (max-voronov) [reply src/components/Slider.js]: Thank you both for the review :)

Here is the requested change: ecd737a5acb1ebdc751b63c51717c93df0eae0f2

**Review by huyenltnguyen — APPROVED** [review]
@max-voronov LGTM! Thanks for the quick change and for your contributions 👍

I'll let Ahmad sign this off

**Pull request merged**